Client/multiplayer_sa/multiplayersa_init.cpp

```diff
@@ -10,9 +10,7 @@
  *****************************************************************************/
 
 #include "StdInc.h"
-#include <net/CNet.h>
 #define DECLARE_PROFILER_SECTION_multiplayersa_init
-#include "profiler/SharedUtil.Profiler.h"
```

Comment:
This is necessary for the profiler, keep it please.

Reply:
this is also unused so it can be removed, "#define DECLARE_PROFILER_SECTION_multiplayersa_init" probably too

Reply:
Profiler integration is done on the build server, don't ask me why, but it's there. Please keep the lines.